sdk/storage/azfile/file/client.go

```diff
@@ -326,6 +326,17 @@ func (f *Client) ListHandles(ctx context.Context, options *ListHandlesOptions) (
 	return resp, err
 }
 
+// CreateHardLink operation returns a list of open handles on a file.
+// For more information, see https://learn.microsoft.com/en-us/rest/api/storageservices/list-handles.
+func (f *Client) CreateHardLink(ctx context.Context, options *CreateHardLinkOptions) (CreateHardLinkResponse, error) {
+	if options == nil {
+		return CreateHardLinkResponse{}, errors.New("targetFile cannot be nil")
```

Comment:
If target file is mandatory parameter, it should be in the method argument. In this case, the check will fail if user passes non-nil options where target file is empty string.